tests/src/test/resources/deviceregistry-mongodb/application.yml

```diff
@@ -5,6 +5,17 @@ hono:
   healthCheck:
     insecurePortBindAddress: 0.0.0.0
     insecurePort: ${vertx.health.port}
+  messaging:
+    name: 'Hono Device Registry MongoDb'
+    host: ${hono.amqp-network.host}
+    port: 5673
+    amqpHostname: hono-internal
+    keyPath: /etc/hono/certs/device-registry-key.pem
+    certPath: /etc/hono/certs/device-registry-cert.pem
+    trustStorePath: /etc/hono/certs/trusted-certs.pem
+    linkEstablishmentTimeout: ${link.establishment.timeout}
+    flowLatency: ${flow.latency}
+    requestTimeout: ${request.timeout}
```

Comment:
This configuration block isn't needed by the changes in this PR, right? In fact I think it shouldn't be needed.
Therefore the block could be omitted here for now and added with the followup PRs where integration tests for the auto-provisioning feature get added.
Same for the other device registry yaml.

Reply:
Good catch! Two less files changed for this PR! 🎉